autoprotocol/container_type.py

```diff
@@ -585,6 +585,46 @@ def row_count(self):
     cat_no="22-387-031"
 )
 
+#:
+V96CC = ContainerType(
+    name="96-well cell culture multiple well plate, V bottom",
+    well_count=96,
+    well_depth_mm=10.668,
+    well_volume_ul=Unit(200.0, "microliter"),
+    well_coating=None,
+    sterile=True,
+    is_tube=False,
+    cover_types=[
+        "standard",
+        "universal",
+        "low_evaporation",
+        "ultra-clear",
+        "foil"
+    ],
+    seal_types=None,
+    capabilities=[
+        "dispense",
+        "spin",
+        "seal",
+        "unseal",
+        "liquid_handle",
+        "cover",
+        "echo_dest",
+        "spectrophotometry",
+        "image_plate",
+        "incubate",
+        "uncover",
+        "dispense-destination",
+        "envision"
+    ],
+    shortname="96-well-v-bottom",
+    col_count=12,
+    dead_volume_ul=Unit(75.0, "microliter"),
```

Comment:
this seems high, but it could be accurate. Being conservative isn't a bad thing here I supose